content/sections/you-can.md

```diff
@@ -0,0 +1,42 @@
+## With Arkouda, you can...
+
+{{< block >}}
+### Make the most of your hardware
+
+{{< graph "scaling" >}}
+
+Source: [Arkouda `argsort` Benchmark](https://github.com/Bears-R-Us/arkouda/tree/master/runs/ronawho-2022-10-03)
+
+Hardware Configuration: HPE SGI 8600:
+* EDR-100 InfiniBand network
+* 40-core Cascade Lake (dual-socket Intel Xeon Gold 6242R)
+* 192 GB RAM
+{{< /block >}}
+
+{{< block >}}
+### Do Exploratory Data Analysis (EDA) on large to massive datasets
```

Comment:
Maybe:

```suggestion
### Do Exploratory Data Analysis (EDA) on massive datasets
```

Reply:
I will leave the remaining wordsmithing and announcement changes to future PRs.